src/main/java/com/starkinfra/StaticBrcode.java

```diff
@@ -59,18 +59,17 @@ public final class StaticBrcode extends Resource {
      * @param name [string]: receiver's name. ex: "Tony Stark"
      * @param keyId [string]: receiver's Pixkey id. ex: "+5541999999999"
      * @param city [string]: receiver's city name. ex: "Rio de Janeiro"
-     * @param amount [long]: positive integer that represents the amount in cents of the resulting Pix transaction. ex: 1234 (= R$ 12.34)
-     * @param reconciliationId [string]: id to be used for conciliation of the resulting Pix transaction. ex: "123"
+     * @param amount [Long]: positive integer that represents the amount in cents of the resulting Pix transaction. ex: 1234 (= R$ 12.34)
+     * @param reconciliationId [string, default ""]: This id must have up to 25 alphanumeric digits ex: "ah27s53agj6493hjds6836v49"
```

Comment:
`id to be used for conciliation of the resulting Pix transaction. This id must have up to 25 alphanumeric digits ex: "ah27s53agj6493hjds6836v49"`

Reply:
Trocar nas outras tmb

Reply:
Troquei essa dosstring tanto no StaticBrcode como no DynamicBrcode